packages/braid-design-system/scripts/generateIcons.cts

```diff
@@ -0,0 +1,230 @@
+/* eslint-disable no-console */
+import path from 'path';
+import fs from 'fs-extra';
+import globby from 'globby';
+import cheerio from 'cheerio';
+import { pascalCase } from 'change-case';
+import dedent from 'dedent';
+// @ts-expect-error svgo@3 has types
+import { optimize } from 'svgo';
+// @ts-expect-error svgr@6 has types
+import svgr from '@svgr/core';
+
+const baseDir = path.join(__dirname, '..');
+const iconComponentsDir = path.join(baseDir, 'src/lib/components/icons');
+
+const relativeToProject = (p: string) => path.relative(baseDir, p);
+
+const componentTemplate = ({ template }: any, opts: any, { componentName, jsx }: any) => {
+  const code = `
+    import React from 'react';
+    NEWLINE
+    import type { SVGProps } from '../SVGTypes';
+    NEWLINE
+    export const COMPONENT_NAME = ({ title, titleId, ...props }: SVGProps) => COMPONENT_JSX;
+  `;
+
+  const reactTemplate = template.smart(code, {
+    plugins: ['react', 'typescript'],
+  });
+
+  return reactTemplate({
+    COMPONENT_NAME: componentName,
+    COMPONENT_JSX: jsx,
+    NEWLINE: '\n',
+  });
+};
+
+const svgrConfig = {
+  svgProps: {
+    focusable: 'false',
+    fill: 'currentColor',
+    width: 16,
+    height: 16,
+  },
+  replaceAttrValues: { '#000': 'currentColor' },
+  template: componentTemplate,
+  plugins: ['@svgr/plugin-jsx', '@svgr/plugin-prettier'],
+  titleProp: true,
+};
+
+(async () => {
+  // First clean up any existing SVG components
+  const existingComponentPaths = await globby('*/*Svg.tsx', {
```

Comment:
This glob was previously scoped to the `iconComponentsDir`, was there a reason this was made more broad?

Reply:
Still is; see the line below.